code/modules/mob/living/simple_animal/bot/mulebot.dm

```diff
@@ -722,9 +722,12 @@
 	if(wires.is_cut(WIRE_AVOIDANCE)) // usually just bumps, but if the avoidance wire is cut, knocks them over.
 		if(iscyborg(L))
 			visible_message("<span class='danger'>[src] bumps into [L]!</span>")
-		else if(L.Knockdown(8 SECONDS))
-			log_combat(src, L, "knocked down")
-			visible_message("<span class='danger'>[src] knocks over [L]!</span>")
+		else
+			L.Knockdown(4 SECONDS)
+			src.turn_off()
+			addtimer(CALLBACK(src, .proc/turn_on), 80)
```

Comment:
use timer defines

Reply:
Can you give me a example for a timer define? Relativiely new to byond code

Reply:
```suggestion
			addtimer(CALLBACK(src, .proc/turn_on), 8 SECONDS)
```
This is a timer define. Feel free to ask for help in our coder channel.